### DISCUSSION TITLE
Read both board & NTC temp

### DISCUSSION THREAD

**Discussion by zt4** [PiSupply/PiJuice#754] [Q&A]
Hi

Is there a way to read both the board temperature and the NTC battery temperature from the Python script (or elsewhere)?

I can read the currently configured temperature sensor via GetBatteryTemperature() and I can change the sensor between NTC and ON_BOARD via SetBatteryTempSenseConfig(config). Just wondering if there is a way to read both values at once without changing the config all the time.

Thanks!

**Comment by tvoverbeek** [comment]
Sorry, not possible with the current firmware.
